packages/mgt-element/src/mock/MockGraph.ts

```diff
@@ -74,13 +75,15 @@ class MockMiddleware implements Middleware {
    */
   private _nextMiddleware: Middleware;
 
-  private static _baseUrl;
+  private static _baseUrl: string;
+
+  private static _session: Session = new Session();
```

Comment:
We should likely be testing that session storage can be used: https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#feature-detecting_localstorage

Perhaps calling the test and assigning the _session property inside the constructor for this object, and then using optional chaining when attempting to access methods on `this._session`?